pages/common/elm.md

```diff
@@ -0,0 +1,15 @@
+# elm
+
+> Run and manage programs in Elm programming language.
+
+- Run interactive elm shell (repl):
```

Comment:
Should `elm` be capitalised here, or should it be lowercase in the description?

Reply:
Thanks, I think capitalising here is better